src/Microsoft.Data.Sqlite.Core/Microsoft.Data.Sqlite.Core.csproj

```diff
@@ -17,7 +17,7 @@ Microsoft.Data.Sqlite.SqliteException
 Microsoft.Data.Sqlite.SqliteFactory
 Microsoft.Data.Sqlite.SqliteParameter
 Microsoft.Data.Sqlite.SqliteTransaction</Description>
-    <TargetFrameworks>net451;netstandard1.2</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;netstandard1.4</TargetFrameworks>
```

Comment:
Because it's being replaced with .NET Standard 2.0, which is supported by .NET 4.6.1.